UploadAndAnnotateImages/force-app/main/default/lwc/imageCapture/imageCapture.js

```diff
@@ -0,0 +1,373 @@
+/* eslint-disable no-await-in-loop */
+import { LightningElement, api, track } from 'lwc';
+import { createRecord } from 'lightning/uiRecordApi';
+import { processImage } from 'lightning/mediaUtils';
+import getContentDocumentId from '@salesforce/apex/ImageCaptureController.getContentDocumentId';
+import getContentVersionId from '@salesforce/apex/ImageCaptureController.getContentVersionId';
+import createContentDocumentLink from '@salesforce/apex/ImageCaptureController.createContentDocumentLink';
+import { log, IMAGE_EXT, isNullOrEmpty, waitMillis, ToastTypes } from "c/utilsImageCapture";
+
+export default class ImageCapture extends LightningElement {
+
+    // This allows the component to be placed on a record page, or other record 
+    // context, and receive the record's ID when it runs
+    @api
+    recordId;
+
+    @api
+    objectApiName;
+
+    @track
+    allImagesData = [];
+
+    compressionOptions = {
+        compressionEnabled: true,
+        resizeMode: 'contain',
+        resizeStrategy: 'reduce',
+        targetWidth: 2048,
+        targetHeight: 2048,
+        compressionQuality: 0.75,
+        imageSmoothingEnabled: true,
+        preserveTransparency: false,
+        backgroundColor: 'white'
+    };
+
+    nextId = 0;
+
+    isReading = false;
+    selectedImageInfo;
+
+    get isImageSelected() {
+        return this.selectedImageInfo != null;
+    }
+
+    isUploading = false;
+    toastType = null;
+    numPhotosToUpload = 0;
+    numSuccessfullyUploadedPhotos = 0;
+
+    get numFailedUploadPhotos() {
+        return this.numPhotosToUpload - this.numSuccessfullyUploadedPhotos;
+    }
+
+    get shouldShowToast() {
+        return this.toastType == null ? false : true;
```

Comment:
Nit: you could simplify this by taking out the ternary in favor of `return this.toastType != null;`

Reply:
Right, thanks!